cloud-native-runtimes/install-cnrt.hbs.md

```diff
@@ -4,17 +4,15 @@ This document describes how to install Cloud Native Runtimes
 from the Tanzu Application Platform package repository.
 
 >**Note:** Use the instructions on this page if you do not want to use a profile to install packages.
-Both the full and light profiles include Cloud Native Runtimes.
+The `full`, `iterate`, and `run` profiles include Cloud Native Runtimes.
 For more information about profiles, see [About Tanzu Application Platform components and profiles](../about-package-profiles.md).
```

Comment:
This `about-package-profiles.md` link is dead. I think we want: [about-package-profiles.hbs.md](https://github.com/pivotal/docs-tap/blob/main/about-package-profiles.hbs.md)